frontend/src/global_styles/content/_table.sass

```diff
@@ -33,13 +33,13 @@
     opacity: 0
 
 $input-elements: input, 'input.form--text-field', select, 'select.form--select', '.form--field-affix', 'a.button'
-$toolbar-height: 80px
+$toolbar-height: 160px
```

Comment:
@bsatarnejad Unfortunately, increasing the height and then adding it to the container height does again cause the double scrollbars which we wanted to get rid of. Where does the 160px come from?

Reply:
Yes it is wrong, i just tested it, i should undo it.

Reply:
@bsatarnejad I pushed a suggestion to fix the dropdown issue and tested it on mobile screen sizes and in Browsers Chrome, Firefox, Edge and Safari. Maybe you could test this solution, too. If it's satisfying, maybe you could resolve the conflicting file and rebase on the dev. If the tests are green then, I think it will be good to merge.

Maybe @oliverguenther can have a look at it and merge it then?